package.json

```diff
@@ -8,6 +8,7 @@
   "repository": "meandavejustice/min-vid",
   "main": "index.js",
   "author": "meandave",
+  "updateURL": "https://testpilot.firefox.com/files/min-vid/updates.json",
```

Comment:
`updateURL` needs to be `path/to/update.rdf`
[example from TxP addon](https://github.com/mozilla/testpilot/blob/master/addon/package.json#L19)